### PR TITLE
Fix broken tests

### PR DESCRIPTION
**Pull request by hyvs**

### CODE REVIEW

File: tests/dependencies.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
     # Goss
     - name: Goss
-      raw: "{{ 'echo \"' ~ item|to_yaml ~ '\"|goss -g - validate' }}"
+      raw: "{{ 'echo \"' ~ item|to_yaml ~ '\"|goss validate' }}"
```
Comment (nervo) [inline tests/dependencies.yml]: This one is related to a recent goss update

**Comment by nervo** [comment]
@hyvs how about this pr ?

File: tests/dependencies.yml (hunk 1 above)
Comment (hyvs) [reply tests/dependencies.yml]: Should I revert this change ?

Comment (nervo) [reply tests/dependencies.yml]: This one, yes, for sure. 
I can not say for the others

**Pull request merged**